src/app/events/bounty/messageCreateOnBountyBoard.ts

```diff
@@ -20,7 +20,7 @@ export default async (message: Message): Promise<any> => {
 	const dbBountyResult: BountyCollection = await dbCollection.findOne({
 		_id: new mongo.ObjectId(bountyId),
 	});
-	const guildMember: GuildMember = message.guild.member(dbBountyResult.createdBy.discordId);
+	const guildMember: GuildMember = message.guild.members.cache.get(dbBountyResult.createdBy.discordId);
```

Comment:
@amaredeus, is there a preference of `.cache.get` over `.fetch` in this case? Has the underlying property of `fetch` changed to no longer check the cache first?

Reply:
hmmm I was following the example for upgrading that section of code https://discordjs.guide/additional-info/changes-in-v13.html#guild

```
- guild.member(user);
+ guild.members.cache.get(user.id)
```

I don't think the underlying logic has changed.. let me update this to fetch instead of cache